tabbycat/motions/views.py

```diff
@@ -157,7 +157,15 @@ def post(self, request, *args, **kwargs):
         round.save()
         self.log_action()
 
-        messages.success(request, self.message_text)
+        if round.motion_set.all().exists():  # Means that the round has at least one motion associated with it
+            messages.success(request, self.message_text)
+        elif self.motions_released:  # The (non-existent) motions were released
+            messages.warning(request, _("No motion(s) are set for this round. Are you sure you intended to already "
+                                        "release the motion(s)?"))
+        else:
+            messages.info(request, _("Returned to previous state. Feel free to enter your motion(s) now, they won't be "
+                                     "publicly displayed until you return to this page and release them."))
+
```

Comment:
These messages should be refined (if these are even necessary).

I don't think these are necessary, or should be in addition to the success message, as that shows that the button was successfully pressed, which it was.

Reply:
To resolve, please indicate why these messages should remain.

Reply:
To me, it seems somewhat intuitive that if the user has released the motions in spite of the warning shown above the button they have either a) not seen the warning, in which case these extra messages should catch this or b) has purposefully chosen to release the motions in spite of this (for reasons unbeknownst to me) so they must clearly have some reason to want to release unset motions. In this case, I'd personally find the message "Motions released successfully" to be somewhat confusing and hence the need to have the second messages.

Reply:
So how about appending something like "No motions were set" or "Motions will be released on creation" then? These toast messages should not be interrogative and I feel they too are badly worded. Please check my overall comment for how I'd think it should work.